hoomd/md/methods/methods.py

```diff
@@ -602,17 +704,47 @@ class Langevin(Method):
         kT (hoomd.variant.Variant): Temperature of the
             simulation :math:`[\mathrm{energy}]`.
 
+            .. rubric:: Examples:
+
+            .. code-block:: python
+
+                langevin.kT = 1.0
+
+            .. code-block:: python
+
+                langevin.kT = hoomd.variant.Ramp(A=2.0,
+                                                 B=1.0,
+                                                 t_start=0,
+                                                 t_ramp=1_000_000)
+
         tally_reservoir_energy (bool): When True, track the energy exchange
             between the thermal reservoir and the particles.
-            :math:`[\mathrm{energy}]`.
+
+            .. rubric:: Example:
+
+            .. code-block:: python
+
+                langevin.tally_reservoir_energy = True
 
         gamma (TypeParameter[ ``particle type``, `float` ]): The drag
             coefficient for each particle type
             :math:`[\mathrm{mass} \cdot \mathrm{time}^{-1}]`.
 
+            .. rubric:: Example:
+
+            .. code-block:: python
+
+                langevin.gamma['A'] = 0.5
```

Comment:
Add a comment to explain particle types? Is this too much?
```suggestion
                # set drag for particle type 'A'
                langevin.gamma['A'] = 0.5
```

Reply:
Yes, this is too much. Comments should not restate the line or line(s) of code that follow.